doc/source/abstractions/index.rst

```diff
@@ -0,0 +1,21 @@
+Abstractions
+############
+
+Abstraction in Python is the process of hiding the real implementation
+of an application from the user and emphasizing only usage.
+
+One of the main objectives of PyAnsys libraries is to wrap (encapsulate) 
+data and methods within units of execution while hiding data or parameters 
+in protected variables.
+
+The topics in this section demonstrate how applications and complex services
+expose functionalities that matter to users and hide all else. For example,
+background details, implementation, and hidden states can all be hidden.
```

Comment:
```suggestion
The topics in this section demonstrate how applications and complex services
expose functionalities that matter to users and hide all else (conditional statements, algorithms...). For example,
background details, implementation, and states can all be hidden.
```

Reply:
Edited because we don't want to use elipses (...). Other things we want to avoid are Latin phrases (like i.e. and e.g.).